.circleci/config.yml

```diff
@@ -5,9 +5,9 @@ jobs:
       BASH_ENV: ~/.nvm/nvm.sh
     docker:
       - image: circleci/golang:1.12.13-browsers
-      - image: 0xorg/ganache-cli:latest
+      - image: 0xorg/ganache-cli:istanbul
```

Comment:
Reminder: We might need to change this depending on @dekz's response.

Reply:
Yep, agreed.